CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+# Changelog
+
+## v1.0.0-beta1 (20 Feb 2017)
```

Comment:
Why beta1? v1.0.0-beta instead?
Another option is to go with your strategy of tracking master and we can release a version when we have our 1.0.0 release.
I know this version is for users to play with and send feedback but some of them might start building services on this and pin it to v1.0.0-beta.
I have a slight preference for keeping it at master for now until our 1.0.0 release.